web/src/lib/domain-logic/getLicenseCalendarEvents.ts

```diff
@@ -1,33 +1,53 @@
-import { defaultDateFormat, LicenseCalendarEvent, LicenseData } from "@businessnjgovnavigator/shared";
-import { parseDate } from "@businessnjgovnavigator/shared/dateHelpers";
+import {
+  defaultDateFormat,
+  LicenseCalendarEvent,
+  LicenseData,
+  Licenses,
+  parseDate,
+} from "@businessnjgovnavigator/shared";
+import { LicenseName, taskIdLicenseNameMapping } from "@businessnjgovnavigator/shared/";
 
 export const getLicenseCalendarEvents = (
   licenseData: LicenseData | undefined,
   year: number,
   month?: number
 ): LicenseCalendarEvent[] => {
   const events: LicenseCalendarEvent[] = [];
-  const isMonthDefined = month !== undefined;
-
-  if (licenseData === undefined || licenseData.expirationISO === undefined) {
+  if (licenseData === undefined) {
     return events;
   }
-  const expirationDate = parseDate(licenseData.expirationISO);
-  if (expirationDate.year() === year && (isMonthDefined ? expirationDate.month() === month : true)) {
-    events.push({
-      dueDate: expirationDate.format(defaultDateFormat),
-      licenseEventSubtype: "expiration",
-      calendarEventType: "LICENSE",
-    });
-  }
-  const renewalDate = expirationDate.add(30, "days");
-  if (renewalDate.year() === year && (isMonthDefined ? renewalDate.month() === month : true)) {
-    events.push({
-      dueDate: renewalDate.format(defaultDateFormat),
-      licenseEventSubtype: "renewal",
-      calendarEventType: "LICENSE",
-    });
-  }
 
+  const isMonthDefined = month !== undefined;
```

Comment:
Do we need to restrict what calendar events appear, i.e. only show the ones relevant to industry

Reply:
Spoke to the team; no change is required.